hbp_validation_framework/__init__.py

```diff
@@ -1777,13 +1777,15 @@ def add_model_instance(self, model_id="", alias="", source="", version="", param
             Path to model source code repository (e.g. github).
         version : string
             User-assigned identifier (unique for each model) associated with model instance.
-        parameters : string
-            Any additional parameters to be submitted to model at runtime.
+        description : string, optional
+            Text describing this specific model instance.
+        parameters : string, optional
+            Any additional parameters to be submitted to model, or used by it, at runtime.
 
         Returns
         -------
-        UUID
-            UUID of the model instance that has been created.
+        dict
```

Comment:
Should return a UUID (or a list or UUIDs, although see comment above)

Reply:
Shall update client to return UUID.